Python/bin/tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This file can't be here. This folder is for scripts that are installed as part of the Python module